man/systemd-vmspawn.xml

```diff
@@ -82,6 +84,21 @@
         If yes is specified KVM is always used, and vice versa if no is set KVM is never used.</para>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--qemu-vsock=</option></term>
+
+        <para>Configure whether to use VSock networking.</para>
+        <para>If the option is not specified VSock support will be detected automatically.
+        If yes is specified VSocks are always used, and vice versa if no is set VSocks are never used.</para>
+      </varlistentry>
+
+      <varlistentry>
+        <term><option>--vsock-cid=</option></term>
+
+        <para>Configure vmspawn to use a specific CID for the guest.</para>
+        <para>If the option is not specified the guest will be assigned the lowest free CID.</para>
```

Comment:
please specify numeric range of expected value here

Reply:
well, i have no idea what the allowed range for CID is. All I am asking here is to help the user (i.e. also me ;-)) and say what it is permissible here.

Reply:
I have chosen to clamp it to the range 3 to UINT_MAX - 1 as the rest are reserved and make no sense to assign. I have added a note of this to the man page.

Reply:
Fixed in 7126b0eb12.